internal/assistedserviceiso/assistedserviceiso_api.go

```diff
@@ -54,10 +55,10 @@ func NewAssistedServiceISOApi(objectHandler s3wrapper.API, authHandler auth.Auth
 func (a *assistedServiceISOApi) CreateISOAndUploadToS3(ctx context.Context, params assisted_service_iso.CreateISOAndUploadToS3Params) middleware.Responder {
 	log := logutil.FromContext(ctx, a.log)
 
-	sshPublicKey := params.AssistedServiceIsoCreateParams.SSHPublicKey
-	if sshPublicKey != "" {
-		sshPublicKey = strings.TrimSpace(sshPublicKey)
-		if err := validations.ValidateSSHPublicKey(sshPublicKey); err != nil {
+	sshPublicKeys := params.AssistedServiceIsoCreateParams.SSHPublicKey
```

Comment:
To make sure I'm not missing something, `params.AssistedServiceIsoCreateParams.SSHPublicKey`  will return a new-line separate list of keys, right? Technically just a string, not an array. Correct?

Reply:
Correct.

Reply:
Did we consider using a slice of strings for `SSHPublicKey` instead of a string with multiple keys? (or add a `SSHPublicKeys` field)

 https://github.com/openshift/assisted-service/blob/902a54d507dc4661d0ca2977114dc8500f52ee05/models/assisted_service_iso_create_params.go#L25

Reply:
Yes, but that would break backward compatibility of the API. This way the backward compatibility is not broken and multiple keys is working.